Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Batch.java

```diff
@@ -112,7 +112,7 @@ public String getTitle() {
     }
 
     /**
-     * The method setTitle() can be used to set a batch title. This function is
+     * Can be used to set a batch title. This function is
```

Comment:
Word _Can_ can't be used as imperative so here it is always incorrect.

Reply:
This is not an imperative, but a verb phrase. A verb phrase (VP) is the part of the sentence without the subject (without the noun phrase, NP). If you want to go deeper, see: [phrase structure grammar](https://en.wikipedia.org/wiki/Phrase_structure_grammar).